Dockerfile

```diff
@@ -45,29 +45,17 @@ COPY --from=builder /tmp/ipxe/src/bin/undionly.kpxe /tmp/ipxe/src/bin-x86_64-efi
 
 COPY --from=builder /tmp/esp.img /tmp/uefi_esp.img
 
-COPY ./ironic.conf.j2 /etc/ironic/ironic.conf.j2
+COPY config/ironic.conf.j2 /etc/ironic/
 
-COPY ./runironic-api.sh /bin/runironic-api
-COPY ./runironic-conductor.sh /bin/runironic-conductor
-COPY ./runironic-exporter.sh /bin/runironic-exporter
-COPY ./rundnsmasq.sh /bin/rundnsmasq
-COPY ./runhttpd.sh /bin/runhttpd
-COPY ./runmariadb.sh /bin/runmariadb
-COPY ./configure-ironic.sh /bin/configure-ironic.sh
-COPY ./ironic-common.sh /bin/ironic-common.sh
-COPY ./runlogwatch.sh /bin/runlogwatch.sh
-
-# TODO(dtantsur): remove this script when we stop supporting running both
-# API and conductor processes via one entry point.
-COPY ./runironic.sh /bin/runironic
-
-COPY ./dnsmasq.conf.j2 /etc/dnsmasq.conf.j2
-COPY ./inspector.ipxe.j2 /tmp/inspector.ipxe.j2
-COPY ./dualboot.ipxe /tmp/dualboot.ipxe
+# TODO(dtantsur): remove scripts/runironic script when we stop supporting
+# running both API and conductor processes via one entry point.
+COPY scripts/ /bin/
```

Comment:
Should this be `scripts/*`? This is correct syntax for rsync, but I can't find any indication in the [Dockerfile docs](https://docs.docker.com/engine/reference/builder/#copy) that the COPY directives treats a source with a trailing slash differently, rsync-style.

Reply:
from [COPY Docker docs](https://docs.docker.com/engine/reference/builder/#copy):
"If <src> is a directory, the entire contents of the directory are copied, including filesystem metadata."

Reply:
And right after that:

>     Note
> 
>     The directory itself is not copied, just its contents.

Reply:
You know I read that like 5 times, and each time my brain transposed the position of the `not` :facepalm: